base_warn_option/models/warn_option.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import fields, models
+
+
+class WarnOption(models.Model):
+    _name = "warn.option"
+    _description = "Warn Option"
+
+    name = fields.Text(required=True, help="This text will be the warning message.")
```

Comment:
IMHO: Maybe translate?

Reply:
No, because in te field that is going to be copied, is not translatable.